release-notes.hbs.md

```diff
@@ -309,8 +309,8 @@ the generated API spec includes API routes from both `CuratedAPIDescriptor`s.
   binding volumes. This happens automatically after upgrading to 1.7 and will not affect subsequent
   upgrades.
 
-  To avoid failures, eqnsure that you have sufficient Kubernetes resources in your clusters to support
-  this pod rollout.
+  Affected workloads and pods will be updated at the same time. To avoid failures, ensure that you have
```

Comment:
**Nitpick:** "Concurrently" might be the better term here. Also, I don't think we technically update `workloads` but only the underlying `pods`.

```suggestion
  Affected pods will be updated concurrently. To avoid failures, ensure that you have
```